images/openwisp_openvpn/openvpn_utils.sh

```
@@ -0,0 +1,93 @@
+	if [ -z "$UUID" ]; then
+		export UUID=$(get_redis_value "openwisp_default_vpn_uuid")
+		export KEY=$(get_redis_value "openwisp_default_vpn_key")
+		export CA_UUID=$(get_redis_value "openwisp_default_vpn_ca_uuid")
+	fi
+}
+
+function openvpn_config_checksum {
+	export OFILE=$(curl --silent --insecure \
+		${API_INTERNAL}/controller/vpn/checksum/$UUID/?key=$KEY)
+	export NFILE=$(cat checksum)
+}
```

Shellcheck (0.11.0)

[warning] 'function' keyword is non-standard. Use 'foo()' instead of 'function foo'.

(SC2113)